### PR TITLE
chore: yarn to pnpm

### PR DESCRIPTION
**Pull request by BlackHole1**
`pack` / `start` / `watch` command execute successed.

### CODE REVIEW

**Comment by erha19** [comment]
@BlackHole1 Thanks for your contribution, but when we want to use `pnpm` as our package manager, we will replace all the projects on the  `opensumi` org.

This PR will be held for some time until our framework core members have a common decision, see https://github.com/opensumi/core/discussions/1572.

File: package.json [hunk 1]
```diff
@@ -50,55 +49,55 @@
     "*.{js,jsx,ts,tsx}": "eslint --fix --quiet"
   },
   "devDependencies": {
-    "@opensumi/ide-addons": "2.18.15",
```
Comment (yantze) [inline package.json:53]: 版本还是保持 2.18.15 ，不用降级

Comment (BlackHole1) [reply package.json:53]: 奇怪，pnpm 在安装过程中，修改了 version 信息。我待会看下

Comment (BlackHole1) [reply package.json]: Fixed

**Pull request merged**